### ISSUE TITLE
i cant get access token from the mobile app

### ISSUE THREAD

**Issue by liquid1** [ronny3050/phone-notification-mirror#25]

**Comment by yomaldyw** [comment]
Yes , i also having a same issue, please give a best solution us .

**Comment by ronny3050** [comment]
I am not sure what the exact problem is?

**Comment by Leonsoyer** [comment]
To get the Access Token, go to pushbullet.com in your browser, log in with your account, go to Settings > Accounts > Access Tokens and click on "Create Access Token".
